spec/support/idv_examples/sp_requested_attributes.rb

```diff
@@ -41,7 +41,7 @@
         expect(page).to have_content t('help_text.requested_attributes.phone')
         expect(page).to have_content '+1 202-555-1212'
         expect(page).to have_content t('help_text.requested_attributes.social_security_number')
-        expect(page).to have_content '666-66-1234'
+        expect(page).to have_content DocAuthHelper::GOOD_SSN
```

Comment:
**question:** is there a reason why you didn't want to assert against `good_ssn` that you set in the `let` block on line 6?

Reply:
good catch, switched to use `good_ssn`